src/components/helpers.js

```diff
@@ -4,6 +4,7 @@ import { Component } from 'preact';
 import { Livechat, useSsl } from '../api';
 import I18n from '../i18n';
 import store from '../store';
```

Comment:
@cauefcr There are a couple of formatting issues introduced here due to the new import. Could you please take care of that. Sidenote: try installing tslint or eslint extension on you vscode - they usually detect these errors and fix them automatically 

![image](https://user-images.githubusercontent.com/34130764/145027267-9023dc73-4c61-476d-9221-ac4b04328eae.png)

Reply:
Sure, i'm working on it now.